lib/solana_rpc_ruby/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SolanaRpcRuby
-  VERSION = '1.3.1'
+  VERSION = '1.3.2'
```

Comment:
Since you're updating Ruby version in this PR, and all dependencies, it definitely shouldn't be a patch, at least minor version upgrade, if not major.